sql/core/src/test/resources/sql-tests/inputs/string-functions.sql

```diff
@@ -46,4 +46,10 @@ FROM (
     encode(string(id + 2), 'utf-8') col3,
     encode(string(id + 3), 'utf-8') col4
   FROM range(10)
-)
+);
+
+-- split function
+select split('aa1cc2ee3', '[1-9]+');
+select split('aa1cc2ee3', '[1-9]+', -1);
+select split('aa1cc2ee3', '[1-9]+', 0);
+select split('aa1cc2ee3', '[1-9]+', 2);
```

Comment:
Likewise, I don't think we should add test cases for all cases. Just one test case and check if they are called fine.